### ISSUE TITLE
Using the hierarchy sunburst chart as a multi level selector

### ISSUE THREAD

**Issue by AurelienBauer** [amcharts/amcharts5#754]
Hello,
I try to use the hierarchy sunburst chart as a selector using the click event, but I did not manage to receive the value that is in the center (focused) when I drill up. The click event callback contains the targeted node, but not the focused one.

**Question**
Is it a way to get the element at the center except if I click on an element who does not have any children?
And as I'm using ReactJS, is it a way to properly bound the chart with react States?

amchart5
Chrome
ReactJS typeScript

**Comment by martynasma** [comment]
How about using series' `dataitemselected` event instead?

```JavaScript
series.events.on("dataitemselected", function(ev) {
  console.log(ev.dataItem.dataContext);
});
```

**Comment by martynasma** [comment]
https://www.amcharts.com/docs/v5/charts/hierarchy/hierarchy-drill-down/#API

**Comment by AurelienBauer** [comment]
Yes It helped thank you ! I had to combine both event, because "dataitemselected" is not trigger when I click on a node with no children.
